src/clients/python/cuda_shared_memory/__init__.py

```diff
@@ -73,6 +73,54 @@ def _raise_error(msg):
     ex = CudaSharedMemoryException(msg)
     raise ex
 
+def serialize_string_tensor(input_tensor):
+    """Creates a shared memory region with the specified name and size.
+
+    Parameters
+    ----------
+    input_tensor : np.array
+        The string tensor to serialize.
+
+    Returns
+    -------
+    serialized_string_tensor : np.array
+        The serialized string tensor.
```

Comment:
Say something about how the serialization is a tensor of type uint8 (or char)

Reply:
Done